packages/react/modules/components/feature-switched/feature-switched.spec.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { shallow } from 'enzyme';
+import FeatureToggled from '../feature-toggled';
+import FeatureSwitched from './feature-switched';
+
+const FeatureComponent = props => <div>FeatureComponent {props.nbr}</div>;
+FeatureComponent.propTypes = { nbr: PropTypes.number.isRequired };
+FeatureComponent.displayName = 'FeatureComponent';
+
+describe('rendering', () => {
+  describe('with a feature match', () => {
+    let wrapper;
+
+    beforeEach(() => {
+      wrapper = shallow(
+        <FeatureSwitched>
+          <FeatureToggled isFeatureEnabled variate="foo">
```

Comment:
Now that it is enabled, I really wonder how widely used this will be without tracking on which Variant is more successful. 

Hmm but then again you can easily wire up your own tracking. Maybe add some documentation about this, the intended scope of the component, and maybe even a simple example in GA on how to implement tracking without bugging the maintainer for this lib about it? 😃

Reply:
Ha, yeah true that. Most adapters have the notion of tracking. E.g. LaunchDarkly with a `client.track("Signed up")`. You define goals there for A/B testing but I haven't used it. https://docs.launchdarkly.com/docs/running-ab-tests

Reply:
And what about `variation`?